jupyterhub/spawner.py

```diff
@@ -495,7 +495,7 @@ def get_args(self):
         """
         args = [
             '--user="%s"' % self.user.name,
-            '--base-url="%s"' % self.user.server.base_url,
```

Comment:
So base_url should be moved inside the server object? (though could be already inside the model, I have to check....)